commands.go

```diff
@@ -715,6 +723,11 @@ func manCMD(rest string, u *User) {
 		return
 	}
 
+	if rest == "prompt" {
+		u.room.broadcast(Devbot, "`prompt <prompt>` sets your prompt\\n\\nYou can use some bash PS1 tags in it.  \\nThe supported tags are:  \\n* \\u: your user name\\n* \\h, \\H: devzat colored like your username\\n* \\t, \\T: the time in your preferred formatting\\n* \\w: the current room\\n* \\W: the current room with #main aliased to ~\\n* \\S: a space character\\n\\nThe default prompt is `\\u:\\S`.")
```

Comment:
use backticks: "`" around this text so less escaping is needed. I don't think you need to have backticks around "prompt \<prompt\>"

Reply:
Will do. Thank for this input.